app/javascript/packages/document-capture/components/in-person-prepare-step.tsx

```diff
@@ -91,6 +92,7 @@ function InPersonPrepareStep({ value }) {
           </Button>
         </div>
       )}
+      <BackButton includeBorder onClick={toPreviousStep} />
```

Comment:
Instead of adding the border support here, can we just use the existing [`PageFooter` component](https://github.com/18F/identity-idp/blob/main/app/javascript/packages/components/page-footer.tsx), which is intended to add the semantics / effect for the bordered separation?

```suggestion
      <PageFooter>
        <BackButton onClick={toPreviousStep} />
      </PageFooter>
```

Reply:
This component is replicating the ERB template version of the Back button, which has some minor stylistic difference with the `PageFooter` component. I think we should leave this as-is for now and reconcile the difference in another story, esp. since it could involve styling tweaks that go beyond the Back button.